### ISSUE TITLE
Moderation feature proposal: official multi-reply for warnings

### ISSUE THREAD

**Issue by gwillen** [LessWrong2/Lesswrong2#610]
This is kind of out-there, I recognize, but:

I think it would be good if a moderator could make an 'official reply' to a bunch of posts at once, with something the following properties:
- It shows up highlighted in a color that indicates it's a moderator reply;
- It shows up as the first response to each post in the thread, OR appended to the text of each such post like an edit;
- It notifies each person in the thread _once_ as a reply, regardless of how many of their messages are in the thread;
- It is collapsible, and is automatically collapsed everywhere except the topmost place it appears, on any given rendering of a page.

It would be good if this could be applied in bulk to:
- All the posts descending from a particular root in a thread;
- All the posts by a specific person(s) in a specific subtree
- Probably other cases I'm not thinking of

This is a wacky feature I have never seen on any forum like this before, and probably has significant drawbacks. But I think it solves an issue that it is hard for a moderator to _get control_ of a large sprawling misbehaving thread, and that a single moderator warning is likely to be overlooked by a lot of participants if a thread has a lot of branches.

This was written after reading Vaniver's post here:

https://www.lesswrong.com/posts/xdwbX9pFEr7Pomaxv/meta-honesty-firming-up-honesty-around-its-edge-cases#prYxKqftQJKLyNpmW

But it's also partly a belated response to some of Duncan Sabien's complaints about moderation. I get the sense that when someone is behaving badly, he wants it to somehow be _made clear_ that they are officially-disapproved and/or community-disapproved, and I think this is hard to do in a framework where a moderator's primary tool to explain themselves is a single post somewhere in a giant tree of discussion, and a user's primary tool is a downvote, but they are unlikely to apply it thoroughly and fairly to an entire tree, especially to posts way down in an subtree that might not be immediately expanded by default e.g. because an ancestor comment has a low score.

I would give a meta-point here, that it would be _surprising_ to me if the tools we've all seen on dozens of forums before are the exact right set to achieve the norms we want here on LW, or even to achieve good moderation of a community in general. (But I also recognize that LW does not have a huge quantity of development resources, as might be necessary to effectively break away from having The Same Tools As Everyone Else, More Or Less.)

**Comment by Raemon** [comment]
Something like this, or at least addressing the underlying concern here, definitely seems pretty valuable.

**Comment by gwillen** [comment]
Yeah, I meant to add a sentence but forgot, to the effect of "this is really a strawman proposal, and I would love to see the same underlying concern explored in other ways." I bet there are other clever ways to do this.
